release/smoke-tests/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.9"
+
+services:
+  data-prepper:
+    restart: unless-stopped
+    image: ${REPOSITORY}/data-prepper:${TAG_NAME}
+    working_dir: /usr/share/data-prepper/
+    volumes:
+      - ./data-prepper/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml
+      - ./data-prepper/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml
+      - ../../shared-config/log4j2.properties:/usr/share/data-prepper/log4j.properties
+    depends_on:
+      - opensearch
+    ports:
+      - "2021:2021"
+
+  opensearch:
+    image: opensearchproject/opensearch:1.0.0
```

Comment:
Please set this to OpenSearch 1.0.1. I believe there were some issues in 1.0.0 that Data Prepper may have issues with.